types/heremaps/index.d.ts

```diff
@@ -1634,6 +1635,11 @@ declare namespace H {
              * @returns {H.map.AbstractMarker} - the marker itself
              */
             setIcon(icon: (H.map.Icon | H.map.DomIcon)): H.map.AbstractMarker;
+
+            /**
+             * This this property ensure that the marker can receive drag events
```

Comment:
It would be nice if this could follow the JSDoc conventions elsewhere, but I'm not too bothered.

Reply:
Done